src/App.js

```diff
@@ -1,66 +1,106 @@
-import React from 'react';
-import TodoList from "./components/TodoComponents/TodoList"
-import TodoForm from "./components/TodoComponents/TodoForm"
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
-  
+
   state = {
     tasks: [
       {
-        task: 'Organize Garage',
+        phrase: "Organize Garage",
         id: 1528817077286,
-        completed: true
+        completed: false
       },
       {
-        task: 'Bake Cookies',
+        phrase: "Bake Cookies",
         id: 1528817084358,
         completed: false
       }
     ],
-    taskInput:""
-  }
-  
-  changeHandler = e =>{
-    e.preventDefault()
+    taskInput: ""
+  };
+
+  // Takes user input as a value, assigns it as the name 
+  changeHandler = e => {
+    e.preventDefault();
     this.setState({
       [e.target.name]: e.target.value
-    })
+    });
+  };
 
-  }
-  
+  // Adds new task - task is set to taskInput above, id is random, default completed is false
   addTask = e => {
-    e.preventDefault()
-    const newTask ={
-    task: this.state.taskInput,
-    id: Date.now(),
-    completed: false,
-  }
-  this.setState ({
-    tasks: [...this.state.tasks, newTask],
-    taskInput: ""
-  })
+    e.preventDefault();
+    const newTask = {
+      phrase: this.state.taskInput,
+      id: Date.now(),
+      completed: false
+    };
+   
+    this.setState({
+      tasks: [...this.state.tasks, newTask],
+      taskInput: ""
+    });
+  };
+
+  // clear button.  Whatever the state is, switch it
+  clearCompleted = e => {
+    e.preventDefault();
+    this.setState({
+      tasks: this.state.tasks.filter(task => !task.completed)
+    });
```

Comment:
Great job realizing you just need to filter and overwrite the original state data that is being rendered to effectively clear the completed ones.